slicer-test/opengl/CircleCI_Slicer_Docker.cmake

```diff
@@ -41,12 +41,19 @@ set( CTEST_SITE "CircleCI_Slicer" )
 # Follow format for caps and components as given on Slicer dashboard
 set( SITE_PLATFORM "CentOS5" )
 
+# Use SITE_BUILD_TYPE specified by slicer-build-with-test
+execute_process (
+    COMMAND bash -c "grep BUILD_TYPE /usr/src/Slicer-build/Slicer-build/CMakeCache.txt | cut -d '=' -f2"
+    OUTPUT_VARIABLE SITE_BUILD_TYPE
+)
```

Comment:
:+1: In general, it is good to avoid state duplication. Rebuilding and testing...